public/tutorials/building-testing-frontend-app-truffle-3.md

```diff
@@ -1,55 +1,67 @@
 
-### Getting Started
-If you were using Truffle beta 3.0.0-9 or below, **do not immediately upgrade**. Read [these release notes](https://github.com/ConsenSys/truffle/releases/tag/v3.0.2) and the [upgrade guide](http://truffleframework.com/tutorials/upgrading-from-truffle-2-to-3) first.
+Uses [webpack](https://webpack.js.org/) to compile the application's frontend code and move the artifacts into the `build/` folder.
+
+## Getting started
+
+Once you have Truffle installed, run `truffle unbox webpack` in an **empty** directory to pull down the [Truffle Box for this tutorial](/boxes/webpack) for this tutorial. If you're familiar with the [MetaCoin Truffle Box](/boxes/metacoin)), you'll notice your old friends `Metacoin.sol` and `ConvertLib.sol` are there. But now, running `truffle build` does this:
 
-Once you have Truffle 3 installed, run `truffle init webpack` in an **empty** directory to pull down the webpack example for this tutorial [(repository here)](https://github.com/trufflesuite/truffle-init-webpack). If you're coming from Truffle 2, you'll notice your old friends `Metacoin.sol` and `ConvertLib.sol` are still there. But now, running `truffle build` does this:
-```bash
-root@b1c71a2e1a6e:/app# truffle build
+```shell
 Error building:
 
-No build configuration specified. Cant build.
+No build configuration specified. Can't build.
 
 Build failed. See above.
 ```
-That's ok! Truffle 3 is getting out of your way and letting you control the build process. Our new process is specified in `webpack.config.js`. More on this later.
 
-### Compile, migrate, and ...
+That's ok! Truffle is getting out of your way and letting you control the build process. Our new process is specified in `webpack.config.js`. More on this later.
+
+## Compile, migrate, and ...
+
 In order to interact with contracts, we need them deployed on a network! The default network is configured in `truffle.js`:
-```json
-...
+
+```javascript
 networks: {
   development: {
     host: '127.0.0.1',
-    port: 8545,
+    port: 7545,
     network_id: '*' // Match any network id
   }
 }
-...
 ```
+
 Let's get the contracts on the network:
 
 First run `truffle compile`. This will compile the `.sol` contracts into `.json` artifacts (specified in the [`truffle-contract`](https://github.com/trufflesuite/truffle-contract) library). They will appear in `build/contracts/*.json`. Now we can include contracts in our app with a simple `import` or `require` statement:
+
 ```javascript
 // Import our contract artifacts and turn them into usable abstractions.
 import metacoin_artifacts from '../../build/contracts/MetaCoin.json'
 ```
 
-Next run `truffle migrate`. This will deploy the contracts onto the default network running at `127.0.0.1:8545`.
+Next run `truffle migrate`. This will deploy the contracts onto the default network running at `127.0.0.1:7545`. This is the default setting for [Ganache](/docs/ganache/using), though you can change this to use any connection you'd like. 
```

Comment:
This seems to be the only mention of Ganache in this page. Should we have a section about getting Ganache up and running? If that's too much extra effort I'd say go with `truffle develop`.

Reply:
Depends. This was really just about making the tutorial still relevant and on-brand. I do link to the Ganache docs, though I can make that clearer.

Note that the Truffle box uses port 7545 by default. If we want to switch to use Truffle Develop, we'll need to change the box default to use port 9545. I vote for Ganache.